styles/main.scss

```diff
@@ -457,13 +458,26 @@ header {
   margin: 0 auto 16px;
 }
 
-.intro-avatar .image img {
+.intro-avatar .image .wrapper img {
   display: block;
   height: 100%;
   width: 100%;
   object-fit: cover;
 }
 
+.intro-avatar .image .wrapper .editor-icon {
+  position: absolute;
+  width: 39px;
+  height: 39px;
+  border-radius: 50%;
+  background: #ffffff;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  object-fit: none;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
👍

Reply:
@fordelord 

left 50%
is not a magic number, because it shift the object on 50% based on **axis of the wrapper element**
translate -50% is not a magic element
because it shift the element based on **50% of it's own axis**